docs/operation/solutions.rst

```diff
@@ -0,0 +1,50 @@
+Solutions Guide
+===============
+
+To deploy a Solution in a MetalK8s cluster, an utility script is provided.
+This section describe, step by step, how to deploy a Solution using this
+tool.
+This script is located at the root of Metalk8s archive:
+
+  .. code::
+
+    /srv/scality/metalk8s-X.X.X/solutions.sh
+
+
+Import Solution
+***************
+
+First, the Solution must be imported in MetalK8s cluster:
+
+  .. code::
+
+    ./solutions.sh import --archive </path/to/solution.iso>
+
+
+Activate Solution
+*****************
+
+Then, the Solution must be activated:
+
+  .. code::
+
+    ./solutions.sh activate --name <solution-name> --version <solution-version>
+
+Environment Creation
+********************
+
+The Solution will run inside an Environment, to create this environment, use:
+
+  .. code::
+
+    ./solutions.sh create-env --name <environment-name>
+
+Adding Solution to an Environment
+******************************
+
+The Solution must then be added to this Environment:
+
+  .. code::
+
+    ./solutions.sh add-solution --name <environment-name> \
+      --solution <solution-name> --version <solution-version>
```

Comment:
After this command, maybe add a really short section to explain that further operations required to deploy and configure the Solution application itself (e.g. a Zenko cluster), our reader should refer to said Solution's documentation and interact with the Operator or UI we just deployed.

Reply:
done